org.signal.Signal.json

```diff
@@ -0,0 +1,71 @@
+{
+    "app-id": "org.signal.Signal",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "signal",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "signal-desktop",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install apply_extra /app/bin",
+                "install signal.sh /app/bin/signal",
+                "install -Dm644 org.signal.Signal.appdata.xml /app/share/appdata/org.signal.Signal.appdata.xml",
+                "cp /usr/bin/ar /app/bin",
+                "cp /usr/lib/libbfd-*.so /app/lib"
+            ],
+            "sources": [
+                {
+                    "type": "extra-data",
```

Comment:
The project is GPL3, we can redistribute it directly which avoids some problems and simplifies the manifest.

Reply:
Fair point. I will change it tomorrow.